api/docs/http_api_index.md

```diff
@@ -1,13 +1,43 @@
 V1 API Services
 ---------------
 
+The V1 API has been removed. All previous routes can now be found under the
+[Master V3](#Master-V3-HTTP-API) API or the [CA V1](#CA-V1-HTTP-API).
+
+V2 HTTP API
+-----------
+The V2 API has been removed. All previous routes can now be found under the
+[Master V3](#V3-HTTP-API).
```

Comment:
Should this and the V1 section be totally removed - as I have done - or should they be marked as deprecated but left otherwise intact?